testsuite/domain/src/test/resources/host-configs/host-slave-failure.xml

```diff
@@ -82,7 +82,7 @@
 
     <domain-controller>
         <!-- Remote domain controller configuration with a host and port -->
-        <remote host="${jboss.test.host.master.address}" port="9999" security-realm="ManagementRealm" />
+        <remote host="${jboss.test.host.master.address}" port="9999" security-realm="ManagementRealm" ignore-unused-configuration="false"/>
```

Comment:
ServerStartFailureTestCase passes without this change.

It's all about the behavior of the slave and its servers but doesn't add new server configs, so ignoring shouldn't affect it.